packages/playground/src/components/node_selector/TfAutoNodeSelector.vue

```diff
@@ -286,6 +286,14 @@ export default {
 
     const nodeInputValidateTask = useAsync<true, string, [NodeInfo | undefined]>(
       async node => {
+        if (node && node?.rentContractId !== 0) {
+          const { state } = await gridStore.grid.contracts.get({
+            id: node?.rentContractId,
+          });
+          if (state.gracePeriod) {
+            return false as true;
```

Comment:
why we should do that? should we just modify the return type from true to boolean?

Reply:
No, as useAsync should return true.. you can try it.

Reply:
![image](https://github.com/user-attachments/assets/b0723bae-cd92-44f5-91c0-86f3f438e917)
compiled successfully, unable to test the functionality now,, will try again whenever the chain transfer got fixed